src/labkey/security/Container.ts

```diff
@@ -85,6 +85,35 @@ export function deleteContainer(config: DeleteContainerOptions): XMLHttpRequest
     });
 }
 
+export interface RenameContainerOptions extends RequestCallbackOptions {
```

Comment:
After manual testing I realized this isn't fully typed for the payload response. Consider:

```ts
export interface RenameContainerOptions extends RequestCallbackOptions<Container>
```